### ISSUE TITLE
ios no sound nor vibration，What causes it？

### ISSUE THREAD

**Issue by xgzl87** [katzer/cordova-plugin-local-notifications#918]

**Comment by irioja** [comment]
I have the same problem. I use this plugin with phonegap 3.7

**Comment by xgzl87** [comment]
I use this plugin with cordova 6.0.0

**Comment by skotturi** [comment]
I'm also experiencing the same thing, my notification sounds dont play on iOS. They do however play on Android. 

My .mps and .wav files are in www/sounds

The code the generates the notification: 
```
var sound = device.platform == 'Android' ? "file://sounds/whistle.mp3" : "file://sounds/whistle.wav";
console.log('the sound file chosen: ' + sound);
var now = new Date().getTime(),
       _5_sec_from_now = new Date(now + (5 * 1000));

cordova.plugins.notification.local.schedule({
                    text: "Delayed Notification",
                    at: _5_sec_from_now,
                    id: 1,
                    sound: sound
                });
```

the correct sound file is logged out for each platform (mp3 and wave). 
The notification is scheduled and gets triggered on both platforms, its just silent on iOS. 
running the following version:
```
Cordova CLI: 5.4.1
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.1
Ionic Version: 1.2.4
Ionic CLI Version: 1.7.14
Ionic App Lib Version: 0.7.0
ios-deploy version: 1.8.4
ios-sim version: 5.0.6
OS: Mac OS X El Capitan
Node Version: v4.4.2
Xcode version: Xcode 7.3 Build version 7D175
```

**Comment by skotturi** [comment]
I've also tried `cordova-cli 6.1.1` and `cordova-ios 4.1.1` without any luck.

**Comment by skotturi** [comment]
update: ios emulate running 9.3 plays notification sound. iphone and ipad still do not.

**Comment by skotturi** [comment]
FIXED. 

When I took my phone out of the case the "silent button" was pushed....

Both my app and the [example app](https://github.com/katzer/cordova-plugin-local-notifications/tree/example) correctly play ios sound notifications.  

@xgzl87 & @irioja I'm happy to try to help you solve this for your apps as well. I've spent the many hours digging through all the documentation and examples trying many different to get it working.
